src/protocols/ssh/ssh.c

```diff
@@ -57,6 +58,38 @@
 #include <sys/socket.h>
 #include <sys/time.h>
 
+/**
+ * This function generates a prompt to the specified instance of guac_client
+ * for the credential specified in the cred_name parameter, which should
+ * be a valid SSH connection parameter.
+ * 
+ * @param client
+ *     The guac_client object associated with the current connection
+ *     where additional credentials are required.
+ * 
+ * @param cred_name
+ *     The name of the parameter to prompt for in the client.
+ */
+static void guac_ssh_get_credential(guac_client *client, char* cred_name) {
+
+    guac_ssh_client* ssh_client = (guac_ssh_client*) client->data;
+    
+    /* If the client does not support the "require" instruction, just return. */
+    if (!guac_client_supports_require(client))
```

Comment:
Hmmm...I think we should probably revert to prompting via the terminal in this case...

Reply:
Yep.

Reply:
Okay, well, I think I have this working, but it ain't pretty.  Happy to take advice, pointers, or whatever code you might be cooking up...